### ISSUE TITLE
Screen reader is not announcing the name of the links present under the PDF.

### ISSUE THREAD

**Issue by msftedad** [gdh1995/pdf.js#16]
**Attach (recommended) or Link to PDF file here**
[edad (1) (1).pdf](https://github.com/gdh1995/pdf.js/files/14139169/edad.1.1.pdf)

**Configuration**
OS: Window 11 22H2 OS Build 25179.1000
Screen reader: Narrator

**Steps to reproduce the problem**
1.Open this PDF viewer [edad (1) (1).pdf](https://github.com/gdh1995/pdf.js/files/14139169/edad.1.1.pdf)
2.Navigate to 'Tools' button and activate 'Open' button.
3.Select any PDF from file, containing links.
4. Verify whether Screen reader is announcing the name of the links present under the PDF or not.

**What is the expected behavior?** (add screenshot)
Aria-label should be defined with link name only, such as 'Securing information systems', and screen reader should announce the name of links as well along with role such as 'Link Securing information systems'.

**What went wrong?** (add screenshot)
Screen reader is not announcing the name of the links present under the PDF.

Screen reader announces 'Link' only when focus moves to links (Ex. Securing information systems).

**Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension)**
 https://mozilla.github.io/pdf.js/web/viewer.html 

**Attachments**

https://github.com/gdh1995/pdf.js/assets/93735775/842b8611-7656-4147-9cc6-73e64ae76cee

<img width="2021" alt="code snippet-Screen reader is not announcing the label for the links" src="https://github.com/gdh1995/pdf.js/assets/93735775/8ba962fd-5821-4461-8cb0-b91123027635">

**Comment by msftedad** [comment]
Hi team, Could you please provide the tentative fix ETA for this issue?

**Comment by msftedad** [comment]
Hi @gdh1995, Any update on this issue? Please provide the ETA to fix this issue.
